pandas/core/indexing.py

```diff
@@ -1595,6 +1595,15 @@ def _get_slice_axis(self, slice_obj, axis=0):
         else:
             return self.obj.take(slice_obj, axis=axis, convert=False)
 
+    def _get_list_axis(self, key_list, axis=0):
+
```

Comment:
so ideally add a Paramaters/Returns section (I know we don't have docs for other of the internal functions, but have to start :))

yes  but be clear that this is a posiitional indexer